.github/workflows/test.yml

```diff
@@ -214,6 +218,9 @@ jobs:
         with:
           prefix-key: "v1-${{ runner.os }}-rust-cache"
           shared-key: ${{ matrix.e2e-type }}
+          cache-all-crates: "true"
+          cache-provider: "buildjet"
```

Comment:
curious if this is needed in other places too? iiuc there are a bunch of different rust cache steps in this file and in rust.yml

Reply:
not for now imo, the goal here is to have the rust caches split across providers - so with this PR:

buildjet:
- e2e-cosmwasm
- e2e-non-cosmwasm

gha:
- rust.yml test-rs
- rust.yml lint-rs
- cli-e2e